DisplaySettings/DisplaySettings.cpp

```diff
@@ -289,7 +289,31 @@ namespace WPEFramework {
             LOGINFO("Starting the timer");
             m_timer.start(RECONNECTION_TIME_IN_MILLISECONDS);
 
-            InitAudioPorts();
+            JsonObject joGetParams;
+            JsonObject joGetResult;
+            joGetParams["params"] = JsonObject();
+            std::string getPowerStateInvoke = "org.rdk.System.1.getPowerState";
+            auto thunderController = getThunderControllerClient();
```

Comment:
Rename getThunderControllerClient to something like getSystemPlugin. This would be similar to getHdmiCecSinkPlugin().